src/types/types.d.ts

```diff
@@ -141,3 +147,27 @@ interface APITweet {
 
   twitter_card: 'tweet' | 'summary' | 'summary_large_image' | 'player';
 }
+
+interface APIUser {
```

Comment:
Perhaps we can merge APIAuthor with APIUser and expose more properties with the Status API as well?

Reply:
I was thinking about that, but I'm not sure if that additional info is really necessary, as there's a lot of extra info that would be returned, plus it'd add another api call that _could_ fail

Reply:
Like, we could have the user endpoint user type extend the other type? It's just that one has a lot more fields like verification, birthday, etc

Not sure if it's worth the extra API call to have it all on one type